sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.getAllInProgressApplications.e2e-spec.ts

```diff
@@ -0,0 +1,204 @@
+      });
+    // Student 2 has an in progress with SABC application offering change request for College F.
+    const inProgressWithSABCApplicationOfferingChange =
+      await saveFakeApplicationOfferingRequestChange(
+        db,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
+          },
+        },
+      );
+    // Student 3 has an in progress with student application offering change request with a different institution.
+    const inProgressWithStudentApplicationOfferingChangeWithDifferentInstitution =
+      await saveFakeApplicationOfferingRequestChange(db);
+    // Student 4 has an completed application for College F, that has an approved application offering change request.
+    const approvedApplicationOfferingChange =
+      await saveFakeApplicationOfferingRequestChange(
+        db,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.Approved,
+          },
+        },
+      );
+
+    const applicationApprovedApplicationOfferingChange =
+      approvedApplicationOfferingChange.application;
+    const applicationInProgressWithStudentApplicationOfferingChangeWithDifferentInstitution =
+      inProgressWithStudentApplicationOfferingChangeWithDifferentInstitution.application;
+    const applicationWithInProgressWithStudentApplicationOfferingChange =
+      inProgressWithStudentApplicationOfferingChange.application;
+    const applicationWithInProgressWithSABCApplicationOfferingChange =
+      inProgressWithSABCApplicationOfferingChange.application;
+
+    applicationApprovedApplicationOfferingChange.student.user.firstName =
+      "Ministry Offering Change Test 1";
+    applicationInProgressWithStudentApplicationOfferingChangeWithDifferentInstitution.student.user.firstName =
+      "Ministry Offering Change Test 2";
+    applicationWithInProgressWithStudentApplicationOfferingChange.student.user.firstName =
+      "Ministry Offering Change Test 3";
+    applicationWithInProgressWithSABCApplicationOfferingChange.student.user.firstName =
+      "Ministry Offering Change Test 4";
+
+    await db.user.save([
+      applicationApprovedApplicationOfferingChange.student.user,
+      applicationInProgressWithStudentApplicationOfferingChangeWithDifferentInstitution
+        .student.user,
+      applicationWithInProgressWithStudentApplicationOfferingChange.student
+        .user,
+      applicationWithInProgressWithSABCApplicationOfferingChange.student.user,
+    ]);
+
+    const endpoint = `/aest/application-offering-change-request/in-progress?page=0&pageLimit=10&sortField=status&sortOrder=${FieldSortOrder.DESC}&searchCriteria=Ministry Offering Change Test`;
```

Comment:
Shouldn't the default sort order be ascending? and also for the default sort order the ordering should have in progress with SABC first right. Because if we use desc as default sort order then older application offering changes gets pushed to the end of the list which is not desired.

![image](https://github.com/bcgov/SIMS/assets/54600590/b5a79410-7fcd-4058-90b1-0f87f1e49082)

Reply:
I changed that to always sort by "applicationOfferingChangeRequest.createdAt" ASC when sorting by status.